elephant/test/test_pandas_bridge.py

```diff
@@ -235,23 +235,23 @@ def test__spiketrain_to_dataframe__parents_empty(self):
         self.assertEqual(keys, res8.columns.names)
 
         for value, level in zip(values, res0.columns.levels):
-            assert_index_equal(value, level)
+            assert_index_equal(value, level, False, False)
```

Comment:
I will refactor this. `exact=False` may be set by default already. I have to check.